BookingsApi/BookingsApi.Infrastructure.Services/Publishers/PublisherHelper.cs

```diff
@@ -30,5 +30,16 @@ public static IEnumerable<Participant> GetNewParticipantsSinceLastUpdate(VideoHe
             }
             return newParticipants;
         }
+
+        public static IEnumerable<Participant> GetAddedParticipantsSinceLastUpdate(VideoHearing videoHearing)
```

Comment:
Will this effect the users added at at the same time the hearing is created? Not sure if that is important but with the process speed and the milliseconds trimmed, the new user and hearing could all be created within a second. If that's not an issue then no worries, ignore my comment.

Reply:
No, created and updated date times are set in the hearing domain model. Hearing needs to be created before any update can be made.